manual.asc

```diff
@@ -513,6 +513,21 @@ You can also manually import .apkg files as follows:
  * Choose the apkg file you just copied to your device when prompted
  * Tap OK
  
+== Importing Anki Databases (.anki2)
+ 
+AnkiDroid does not yet support directly importing Anki database (`.anki2`) files. A database import will replace your collection with the contents the provided file. If you want to perform a full replacement, you should ideally import an exported `collection[.apkg/.colpkg]`, as this also includes media data.
```

Comment:
I don't see us ever supporting this, re-wording to reflect

```suggestion
AnkiDroid does not support directly importing Anki database (`.anki2`) files. A database import will replace your collection with the contents the provided file. If you want to perform a full replacement, you should import a `collection[.apkg/.colpkg]` created in the app with the export function.
```

Reply:
I'd like to (not a priority - might never be) - an `anki2` file import is effectively the same as a `collection.apkg` import without media. We've already got the code to do it, and it makes for a slightly nicer user experience.

Reply:
Whereas I really don't want to. Having multiple ways to import/export, one of which is getting into internals, violates the packaging we strive for and removes the possibility that on import/export we can do things like check if they have scheduler compatibility, or perhaps alter things for coming libanki changes etc etc. I do not want to go anywhere near it - I want to preserve the packages as they exist for import export